proposals/0079-allow_commission_decrease_at_any_time.md

```diff
@@ -0,0 +1,88 @@
+---
+simd: '0079'
+title: Allow Commission Decrease at Any Time
+authors:
+  - Bryan Ischo (bryan@ischo.com)
+category: Standard
+type: Core
+status: Draft
+created: 2023-10-26
+feature: https://github.com/solana-labs/solana/issues/29361
+---
+
+## Summary
+
+The commission_updates_only_allowed_in_first_half_of_epoch feature disallows
+commission decrease in the second half of epoch. Given that the purpose of this
+feature was to prevent 'rug pulls' which are accomplished by increasing
+commission at the end of epoch and then decreasing commission at the beginning
+of next epoch, disallowing decreases during the second half of epoch is
+unnecessary.
+
+A feature gate must be added to support this SIMD as all validators' vote
+programs must treat commission change instructions the same or else consensus
+will diverge.
+
+## Motivation
+
+Some validator operators may need to decrease commission in order to satisfy
+their own operational criteria.
+
+As an example, a validaor operator may have a policy whereby any error that
+results in reduced stake account rewards for the epoch, will result in the
+operator choosing to reduce commission to 0% for that epoch to ensure that
+stake accounts are not disadvantaged by that error.  Not being allowed to do
+this in the second half of an epoch is a problem because it would prevent that
+commission change until the next epoch, which will not allow this policy to
+take effect for stake accounts which were de-activating during that epoch.
+
+Allowing commission changes during the second half of epochs also enables
+"reverse rug pulls" where a validator has commission set to 100% during the
+majority of an epoch but reduces it to 0% across the epoch boundary.  It's not
+clear why some validator operators choose to do this, but it does not seem
+harmful, and there does appear to be motivation for some validators to do this
+as it's been done numerous times in the past.
+
+## Alternatives Considered
+
+No alternatives were considered.
+
+## New Terminology
+
+Reverse Rug Pull: Setting commission to 100% for most of an epoch, and to 0%
+only across epoch boundaries.  This has the effect of discouraging third party
+stake to a validator while still charging 0% commission.  This is not
+considered to be a technique harmful to stakers.
```

Comment:
I didn't realize that the purpose of this section was to introduce "official" terminology (not even sure what "official" means here) - I thought it was just to introduce terms that are used or might be used in discussion of the feature that are not otherwise commonly known.  However, I will remove this section as it doesn't accomplish much in this SIMD (was just trying to provide some background).